integration-tests/typescript-koa/src/api.github.com.yaml/models.ts

```diff
@@ -743,7 +743,7 @@ export type t_code_scanning_codeql_database = {
 }
 
 export type t_code_scanning_default_setup = {
-  languages?: "javascript" | "python" | "ruby"[]
+  languages?: ("javascript" | "python" | "ruby")[]
```

Comment:
Previously we erroneously made this property accept some `javascript`/`python` or an array of the string `ruby` only :facepalm: - with the new helper functions we do a better job of adding `()`